content/accessibility/accessibility-at-github.mdx

```diff
@@ -0,0 +1,48 @@
+
+In many cases, devices that have been designed with accessibility in mind for a portion of the population end up benefiting everyone. Take the [OXO brand's Good Grips line of products](https://www.oxo.com/blog/behind-the-scenes/behind-design-oxos-iconic-good-grips-handles/); although they were developed for people who have arthritis, most people found them easier to use than other product lines.
+
+When you consider accessibility from the beginning of the design process and incorporate it throughout, it becomes part of an established design process that seeks to include everyone, of all abilities, and allows everyone to achieve more.
+
+**Note:** When reading through accessibility information, you might find the common abbreviation: "a11y". A11y is a shortening of the word "accessibility". It starts with the first letter, "a", uses the number 11 for the eleven characters inbetween the first and last letter, and ends with the last letter, "y".
+
+## What standards you must aim for 
+GitHub aims for [Web Content Accessibility Guidelines (WCAG) 2.1](https://www.w3.org/TR/WCAG21/) AA compliance. This includes all of WCAG 2.0 AA plus additional considerations. 
+
+Currently, [Section 508](https://www.access-board.gov/guidelines-and-standards/communications-and-it/about-the-ict-refresh/overview-of-the-final-rule) (required by law in the United States) follows WCAG 2.0 AA guidelines for compliance. 
+
+## Who should you consider when designing? 
+There are a whole host of disabilities to consider, though the majority will fall into one of these categories: 
+- **Visual:** Anything that deals with sight, or technology that helps someone to see. This includes people who have different types of sight like color blindness.
+- **Cognitive:** Concentration, memory, judgement, problem solving, logic skills. 
+- **Mobility:** Anything that affects movement in a body. 
+- **Hearing:** A spectrum of disabilities related to sound or audio.
+
+Each of these disability categories can be divided into three further sub-categories:
+- **Permanent:** Disability does not go away.
+  - Example: Someone who is deaf
+- **Temporary:** The disability will go away in time.
+  - Example: Someone who has an ear infection
+- **Situational:** The environment someone is in plays a huge factor in how they interact with web content. 
+  - Example: Someone who is at a loud sporting event
+
+For an informative list on real-life disability situations, check [An Alphabet of Accessibility Issues](https://the-pastry-box-project.net/anne-gibson/2014-july-31), published in The Pastry Box Project.
+
+Microsoft has created a downloadable PDF that looks into the area of inclusive design from a people perspective, with several informative examples. Navigate to the ‘Inclusive 101’ toolkit on the main page of [Microsoft’s Inclusive Design website](https://www.microsoft.com/design/inclusive/).
```

Comment:
This area could be condensed and listed as "Additional resources".

Reply:
I left these mentions in this section because they're both related to who we should consider while designing. Is it okay to keep them in this section and then do something like: 

### Additional resources
- [An Alphabet of Accessibility Issues](https://the-pastry-box-project.net/anne-gibson/2014-july-31), published by The Pastry Box Project, has an informative list on real-life disability situations.
- Microsoft has created a downloadable PDF that looks into the area of inclusive design from a people perspective, with several informative examples. Navigate to the ‘Inclusive 101’ toolkit on the main page of [Microsoft’s Inclusive Design website](https://www.microsoft.com/design/inclusive/).

Reply:
looks ok for me, @ashygee ?